packages/@okta/vuepress-site/conductor.yml

```diff
@@ -917,16 +917,6 @@ redirects:
     to: /docs/guides/protect-your-api/aspnet/next-steps/
   - from: /guides/protect-your-api/aspnet/require-authentication/index.html
     to: /docs/guides/protect-your-api/aspnet/require-authentication/
-  - from: /guides/protect-your-api/aspnetcore/before-you-begin/index.html
```

Comment:
Should we map to aspnetcore3 instead?

Reply:
I'm not sure. I removed links to deleted docs, but did not touch any other, if links to aspnetcore3 existed they are still there

Reply:
They might not be present in this redirection list though.